config/config.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+import (
+	"strings"
+
+	"github.com/asyncapi/event-gateway/kafka"
+)
+
+// App holds the config for the whole application.
+type App struct {
+	Debug       bool
+	AsyncAPIDoc []byte     `split_words:"true"`
+	KafkaProxy  KafkaProxy `split_words:"true"`
+}
+
+// ProxyConfig creates a config struct for the Kafka Proxy.
+func (c App) ProxyConfig() (kafka.ProxyConfig, error) {
+	return c.KafkaProxy.ProxyConfig(c.AsyncAPIDoc, c.Debug)
+}
+
+type pipeSeparatedValues struct {
+	Values []string
+}
+
+func (b *pipeSeparatedValues) Set(value string) error {
```

Comment:
Do we really need that function returns the error? Or do you make it for the future?

Reply:
I'm reminding myself why I don't like this language 😅

Reply:
Embrace the "duck typing" @magicmatatjahu !!!!! 🦆  ⌨️  🚀